app/javascript/components/mentoring/inbox/conversation-list/Conversation.jsx

```diff
@@ -16,30 +17,54 @@ export function Conversation({
   updatedAt,
   url,
 }) {
+  const [referenceElement, setReferenceElement] = useState(null)
+  const [popperElement, setPopperElement] = useState(null)
+  const [showPopper, setShowPopper] = useState(null)
+  const { styles, attributes } = usePopper(referenceElement, popperElement)
+
   return (
-    <tr>
-      <td>
-        <img
-          style={{ width: 100 }}
-          src={trackIconUrl}
-          alt={`icon for ${trackTitle} track`}
-        />
-      </td>
-      <td>
-        <img
-          style={{ width: 100 }}
-          src={menteeAvatarUrl}
-          alt={`avatar for ${menteeHandle}`}
-        />
-      </td>
-      <td>{menteeHandle}</td>
-      <td>{exerciseTitle}</td>
-      <td>{isStarred.toString()}</td>
-      <td>{haveMentoredPreviously.toString()}</td>
-      <td>{isNewIteration.toString()}</td>
-      <td>{postsCount}</td>
-      <td>{dayjs(updatedAt).fromNow()}</td>
-      <td>{url}</td>
-    </tr>
+    <>
+      <tr
+        ref={setReferenceElement}
+        onMouseEnter={() => {
+          setShowPopper(true)
+        }}
+        onMouseLeave={() => {
+          setShowPopper(null)
+        }}
```

Comment:
As we only have a single line and `setShowPopper` returns `void`, this could also be written as:

```suggestion
        onMouseEnter={() => setShowPopper(true)}
        onMouseLeave={() => setShowPopper(false)}
```